### PR TITLE
chore: bump lru and jsonrpsee to latest

### PR DESCRIPTION
**Pull request by liya2017**
**What this PR does / why we need it**:

This PR

**Which issue(s) this PR fixes**:

Fixes #

**Which docs this PR relation**:

Ref #

**Which toolchain this PR adaption**:

No Breaking Change

**Special notes for your reviewer**:

NIL

**CI Description**

| CI Name                                   | Description                                                     |
| ----------------------------------------- | --------------------------------------------------------------- |
| *Chaos CI*                                | Test the liveness and robustness of Axon under terrible network condition    |
| *Cargo Clippy*                            | Run `cargo clippy --all --all-targets --all-features`      |
| *Coverage Test*                           | Get the unit test coverage report                             |
| *E2E Test*                                | Run end-to-end test to check interfaces                         |
| *Code Format*                             | Run `cargo +nightly fmt --all -- --check` and `cargo sort -gwc`     |
| *Web3 Compatible Test*                    | Test the Web3 compatibility of Axon                               |
| *v3 Core Test*                            | Run the compatibility tests provided by Uniswap V3             |
| *OCT 1-5 \| 6-10 \| 11 \| 12-15 \| 16-19* | Run the compatibility tests provided by OpenZeppelin           |

**CI Usage**

> Check the CI you want to run below, and then comment `/run-ci`.

**CI Switch**

- [ ] Chaos CI
- [x] Cargo Clippy
- [ ] Coverage Test
- [x] E2E Tests
- [x] Code Format
- [x] Unit Tests
- [ ] Web3 Compatible Tests
- [ ] OCT 1-5 And 12-15
- [ ] OCT 6-10
- [ ] OCT 11
- [ ] OCT 16-19
- [ ] v3 Core Tests

### CODE REVIEW

File: core/api/Cargo.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@ edition = "2021"
 # async-graphql = { version = "3.0", features = ["tracing"] }
 beef = "0.5"
 
-jsonrpsee = { version = "0.15", features = ["http-server", "macros", "ws-server"] }
+jsonrpsee = { version = "0.16", features = ["client-core", "macros","server"] }
```
Comment (driftluo) [inline core/api/Cargo.toml]: I think we're not using the client-core feature

File: core/executor/Cargo.toml [hunk 2]
```diff
@@ -22,7 +22,7 @@ futures = "0.3"
 hasher = "0.1"
 lazy_static = "1.4"
 log = "0.4"
-lru = "0.9"
+lru = "0.10.0"
```
Comment (KaoImin) [inline core/executor/Cargo.toml]:
```suggestion
lru = "0.10"
```

File: core/storage/Cargo.toml [hunk 3]
```diff
@@ -9,7 +9,7 @@ arc-swap = "1.6"
 futures = "0.3"
 lazy_static = "1.4"
 log = "0.4"
-lru = "0.9"
+lru = "0.10.0"
```
Comment (KaoImin) [inline core/storage/Cargo.toml]:
```suggestion
lru = "0.10"
```

File: core/api/Cargo.toml (hunk 1 above)
Comment (KaoImin) [inline core/api/Cargo.toml]:
```suggestion
jsonrpsee = { version = "0.16", features = ["macros","server"] }
```

**Review by KaoImin — CHANGES_REQUESTED** [review]

**Review by KaoImin — APPROVED** [review]

File: core/api/Cargo.toml (hunk 1 above)
Comment (liya2017) [reply core/api/Cargo.toml]: done

**Pull request merged**